src/wallet/spend.cpp

```diff
@@ -651,13 +651,12 @@ static void DiscourageFeeSniping(CMutableTransaction& tx, FastRandomContext& rng
     }
 }
 
-static bool CreateTransactionInternal(
+static OperationResult CreateTransactionInternal(
```

Comment:
In 30feca7b374256ff62443b2e2f1c65dfa49bb083 "send: move wallet CreateTransaction* flow to use OperationResult/CallResult."

I don't think this needs to be an `OperationResult`. It really can be `CallResult<CTransactionRef>`. The original really should have just returned `CTransactionRef` anyways.